.drone.yml

```diff
@@ -348,6 +348,27 @@ services:
       - sed -i '/^pycti==/d' src/requirements.txt
       - pip3 install -r src/requirements.txt
       - python3 src/worker.py
+  - name: opencti-test-worker
+    image: nikolaik/python-nodejs:python3.11-nodejs20-alpine
+    volumes:
+      - name: cache-node-backend
+        path: /tmp/direct-start-platform/node_modules
```

Comment:
I think it's one dedicated cache per step, you need to add it at the end of drone file.... And node module for worker ?
Maybe it's just no cache at all for this step.